packages/shared/src/lib/contexts/campaigns/interfaces/leaderboard.interface.ts

```diff
@@ -0,0 +1,6 @@
+import { ITideLeaderboardItem, ITideUserPosition } from '@core/tide/interfaces'
+
+export interface ILeaderBoard {
+    board: ITideLeaderboardItem[]
+    userPosition: ITideUserPosition
```

Comment:
we can probably ignore user position here? Not sure how the api even returns anything if they dont know our user